eZ/Bundle/EzPublishCoreBundle/Resources/config/services.yml

```diff
@@ -38,20 +38,55 @@ services:
         class: "%ezpublish.siteaccess.class%"
         arguments: ["%ezpublish.siteaccess.default.name%", 'uninitialized']
 
-    ezpublish.config.resolver.core:
-        public: true # @todo should be private
-        class: "%ezpublish.config.resolver.dynamic.class%"
-        arguments: ["@?logger", "%ezpublish.siteaccess.groups_by_siteaccess%", "%ezpublish.config.default_scope%"]
+    ezpublish.config.resolver.default:
```

Comment:
I would actually prefer not to introduce new `ezpublish`-prefixed services. IMHO there's nothing stopping us from using FQCN service names, per Symfony 3.x recommendation. Step-by step process (evolution) in this case is more important to me than consistency with surrounding code.